vision/getting-started/image_editing.ipynb

```diff
@@ -1,710 +1,1247 @@
 {
```

Comment:
<p>"...you can insert new <strong>content</strong> into an image. "</p><p> </p><p>This description is very unclear and should be more specific. What is meant by "insert new content"?</p><p> </p><p>This might be a good opportunity to show a visual example, rather than just a textual explanation -- just an idea!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/506/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Updated description. 

Re: comments regarding showing visual samples. I agree and I will add them in. However in the interest of not adding more to this already large change, I will add visual samples in the next change.